code/game/objects/effects/overlays.dm

```diff
@@ -32,18 +32,12 @@
 	return QDEL_HINT_PUTINPOOL
 
 /obj/effect/overlay/temp/New()
+	..()
```

Comment:
I'm not questioning that this is needed, but what goes on upstream in the object tree